builtin/logical/consul/path_roles.go

```diff
@@ -50,10 +56,8 @@ and instead be local to the current datacenter.  Available in Consul 1.4 and abo
 			"token_type": {
 				Type:    framework.TypeString,
 				Default: "client",
-				Description: `Which type of token to create: 'client'
-or 'management'. If a 'management' token,
-the "policy" parameter is not required.
-Defaults to 'client'.`,
+				Description: `Which type of token to create: 'client' or 'management'. If
+a 'management' token, the "policy" parameter is not required. Defaults to 'client'.`,
```

Comment:
```suggestion
a 'management' token, the "policy" and "role" parameters are not required. Defaults to 'client'.`,
```

Reply:
Reworded it slightly more to mention both policy doc and the newer policies. Also added the same statement to the web docs, similar to your comment above. https://github.com/hashicorp/vault/pull/14014/commits/aa1901b0cb5e6f64ad77cd8f807b2b51195ac534